lang/levels/standard/text_fre.po

```diff
@@ -0,0 +1,841 @@
+msgstr "Comme vous avez réclamé votre première salle, les héros ont commencé à rassembler leurs troupes et seront bientôt là. Plus les héros sont anéantis, plus les groupes qu'ils formeront seront forts."
+
+#. Grail Run (lv444)
+#: guitext:104
+msgctxt "In-game message"
+msgid "First party has arrived! They seem to go lightly on on you. For now..."
+msgstr "Le premier groupe est arrivée ! Ils semblent aller légèrement sur vous. Pour l'instant..."
+
+#. Grail Run (lv444)
+#: guitext:105
+msgctxt "In-game message"
+msgid "Having seen their previous efforts fail, the Heroes have formed a new group."
+msgstr "Ayant vu leurs efforts précédents échouer, les Héros ont formé un nouveau groupe."
+
+#. Grail Run (lv444)
+#: guitext:106
+msgctxt "In-game message"
+msgid "Heroes have returned, this time with modified creatures to aid them! Careful, as these are definitely tougher than any common dungeon creatures!"
+msgstr "Les héros sont de retour, cette fois avec des créatures modifiées pour les aider ! Attention, car ils sont certainement plus résistants que n'importe quelle créature de base du donjon !"
+
+#. Grail Run (lv444)
+#: guitext:107
+msgctxt "In-game message"
+msgid "Heroes have formed yet another army. If this attempt fails, the Avatar will come and take care of you!"
+msgstr "Les héros ont formé une autre armée. Si cette tentative échoue, l'Avatar viendra s'occuper de vous !"
+
+#. Grail Run (lv444)
+#: guitext:108
+msgctxt "In-game message"
+msgid "It's time to prove yourself, Keeper. Clear the last wave and the land is yours!"
+msgstr "Il est temps de faire vos preuves, Gardien. Effacez la dernière vague et la terre est à vous !"
+
+#. Grail Run (lv444)
+#: guitext:109
+msgctxt "In-game message"
+msgid "This has been a humiliating defeat for the Heroes. Congratulations!"
+msgstr "Cela a été une défaite humiliante pour les héros. Félicitations!"
+
+#. Grail Run (lv444)
+#: guitext:110
+msgctxt "In-game message"
+msgid "You are in possession of a very unusual hero gate located to the east of your dungeon heart. It attracts units from other realms based on your claimed  dungeon rooms. Claim rooms to attract more and stronger units. Barracks=Barbarians, Workshop=Dwarves, Library=Warlocks, Temple=Monks, Training Rooms=Archers. New units arrive in groups but only if you have fewer than 22 minions. Withstand the hero waves, defeat the Avatar and his guards and this land is yours!"
+msgstr "Vous êtes en possession d'une porte de héros très inhabituelle située à l'est de votre coeur de donjon. Il attire des unités d'autres royaumes en fonction de vos salles de donjon revendiquées. Réclamez des chambres pour attirer des unités plus nombreuses et plus fortes. Caserne = Barbares, Atelier = Nains, Bibliothèque = Sorciers, Temple = Moines, Salles d'entraînement = Archers. Les nouvelles unités arrivent en groupe mais seulement si vous avez moins de 22 sbires. Résistez aux vagues de héros, battez l'Avatar et ses gardes et cette terre est à vous !"
```

Comment:
"coeur" -> "cœur" mais seulement si KeeperFX permet les "œ".
"Bibliothèque = Sorciers" -> "Bibliothèque = Mages" le Sorcier étant la créature.

Reply:
It does spawn the creature here, so Warlocks. Is this correct now in the translations?